translations/de.yaml

```diff
@@ -344,3 +344,8 @@ ember-gwr:
 
     modelErrors:
       empty: Keine offenen Pendenzen
+
+    streetSearch:
+      requiredInputs: Bitte geben Sie zuerst Ortsname und PLZ an.
```

Comment:
Will need a FR translation

Reply:
`done`

Reply:
Requested a FR translation, we can update it in a later release.